### PR TITLE
Pydata sphinx theme

### PR DESCRIPTION
**Pull request by rwedge**
Updates the docs to use the pydata sphinx theme

https://featuretools.alteryx.com/en/pydata-sphinx-theme/index.html

### CODE REVIEW

File: dev-requirements.txt [hunk 1]
```diff
@@ -1,17 +1,17 @@
 -r test-requirements.txt
 featuretools-sklearn-transformer >= 0.1.1
-codecov==2.1.0
+codecov==2.1.8
 flake8==3.7.8
 autopep8==1.4.4
-ipython==7.14.0
+ipython==7.17.0
```
Comment (gsheni) [inline dev-requirements.txt:6]: You can further upgrade ipython with the following
```
ipython==7.16.0; python_version=='3.6'
ipython==7.18.1; python_version>'3.6'
```

File: dev-requirements.txt [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-matplotlib==3.2.1
+matplotlib==3.3.1
 nbconvert==5.6.1
-nbsphinx==0.7.0
-Sphinx==3.0.3
-sphinx_rtd_theme==0.4.3
-nlp_primitives>=0.3.0
+nbsphinx==0.7.1
+Sphinx==3.2.1
+pydata_sphinx_theme==0.3.1
```
Comment (gsheni) [inline dev-requirements.txt:13]: A higher theme version, if you want
```
pydata-sphinx-theme==0.4.0
```

Comment (rwedge) [reply dev-requirements.txt]: updated

File: docs/source/getting_started/getting_started_index.rst [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Getting Started
+---------------
+
+How to start working with Featuretools; The main concepts
```
Comment (frances-h) [inline docs/source/getting_started/getting_started_index.rst:4]: Would it make sense to add a link back to the 5 minute quick start here? Something like:
> For a quick introduction to Featuretools, check out our [5 minute quick start guide](https://docs.featuretools.com/en/pydata-sphinx-theme/index.html#minute-quick-start).
>
> How to start working with Featuretools; the main concepts:

Comment (rwedge) [reply docs/source/getting_started/getting_started_index.rst:4]: Good idea

**Review by frances-h — APPROVED** [review]
looks good!

**Pull request merged**